GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/ExtensionsAspNetCoreTests/ConfigTests.cs

```diff
@@ -10,9 +10,9 @@ public class ConfigTests
 
         [Test]
         public void Config_DefaultDate_Returns_DateTime_Now_Plus_Three_Days_And_One_Hour_When_Configured()

```

Comment:
Should we rename this test to `Config_DefaultDate_Returns_DateTime_Now_Plus_917_Days_When_Configured()`?